src/stores/messages/MessagesDataProviderStore.ts

```diff
@@ -280,6 +311,8 @@ export default class MessagesDataProviderStore {
 			query,
 			this.onNextSoftFilterChannelResponse,
 			this.onLoadingError,
+			// eslint-disable-next-line @typescript-eslint/no-empty-function
+			() => {},
```

Comment:
lets make it optional otherwise this looks bad. i'd rather explicitly pass undefined or null if i dont need that function